hack/test-update-storage-objects.sh

```diff
@@ -93,7 +93,7 @@ startApiServer
 
 # Create a pod
 kube::log::status "Creating a pod"
-${KUBECTL} create -f examples/pod.yaml
+${KUBECTL} create -f docs/user-guide/pod.yaml
```

Comment:
yes it runs in shippable. pod.yaml is duplicated in both locations to alleviate build woes cause by #11239. this finishes the migration.